### DISCUSSION TITLE
how to setup the analytics?

### DISCUSSION THREAD

**Discussion by postyy2** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#546] [General]
Hi,

How can I setup anayltics? Seems a little bit confusing.
I don't understand what "just call the analytics method before mining" means

Thank you.

**Comment by Tkd-Alex** [comment]
Follow the readme, there is a clear example.
In your `run.py` write `twitch_miner.analytics` and then `twitch_miner.mine`
